extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesDeployer.java

```diff
@@ -1,6 +1,7 @@
 
 package io.quarkus.kubernetes.deployment;
 
+import static io.quarkus.kubernetes.deployment.Constants.DEPLOYMENT_CONFIG;
```

Comment:
I agree with this change, but since its irrelevant to our goal, I would have it in another pull request.
In general its good to keep PRs as focused as possible.